.gitmodules

```diff
@@ -4,6 +4,9 @@
 [submodule "external/fv3gfs-wrapper"]
 	path = external/fv3gfs-wrapper
 	url = git@github.com:ai2cm/fv3gfs-wrapper.git
+[submodule "external/gt4py"]
```

Comment:
Is this where specific tags would be indicated (e.g., the sha for `v36`)?

Reply:
You'd indicate a tag by checking out that commit in the submodule, and adding it to the repo with `git add external/gt4py` or similar. Checking it out by tag or commit makes no difference.